velox/common/encode/Base64.h

```diff
@@ -101,8 +102,16 @@ class Base64 {
   static size_t
   decode(const char* src, size_t src_len, char* dst, size_t dst_len);
 
- private:
+  static void decodeUrl(
+      const char* src,
+      size_t src_len,
+      char* dst,
+      size_t dst_len,
+      bool pad);
+
   constexpr static char kBase64Pad = '=';
```

Comment:
Does this variable need to be public ? Seems like you moved the private keyword below it.

Reply:
Yes, as I am using this variable `kBase64Pad` in `stringImpl.h` in my function to access it rather than using the '='. 
```
const bool hasPad =
        (*(input.end()) == encoding::Base64::kBase64Pad) ? true : false;
```